drivers/sensor/stm32_temp/stm32_temp.c

```diff
@@ -176,8 +179,8 @@ static const struct stm32_temp_config stm32_temp_dev_config = {
 #else
 	.avgslope = DT_INST_PROP(0, avgslope),
 	.v25_mv = DT_INST_PROP(0, v25),
-	.is_ntc = DT_INST_PROP(0, ntc)
 #endif
+	.is_ntc = DT_INST_PROP_OR(0, ntc, false)
```

Comment:
TBC but I think DT_INST_PROP should be sufficient for a boolean.

Reply:
It apparently isn't, CI was failing before I made this change.